src/Tizen.Multimedia.Remoting/MediaController/MediaController.cs

```diff
@@ -322,7 +274,55 @@ public MediaControlRepeatMode GetRepeatMode()
         }
 
         /// <summary>
-        /// Sends playback command to the server.</summary>
+        /// Requests command to the server.
+        /// </summary>
+        /// <remarks>
+        /// The client can request the server to execute <see cref="PlaybackCommand"/> or <see cref="ShuffleModeCommand"/> or
+        /// <see cref="RepeatModeCommand"/> or <see cref="CustomCommand"/>, <br/>
+        /// and then, the client receive the result of each request(command).
+        /// </remarks>
+        /// <param name="command">A <see cref="Command"/> class.</param>
+        /// <exception cref="InvalidOperationException">
+        ///     The server has already been stopped.<br/>
+        ///     -or-<br/>
+        ///     An internal error occurs.
+        /// </exception>
+        /// <exception cref="ObjectDisposedException">The <see cref="MediaControllerManager"/> has already been disposed of.</exception>
+        /// <since_tizen> 5 </since_tizen>
+        public async Task RequestAsync(Command command)
+        {
+            ThrowIfStopped();
+
+            command.SetServerInfo(Manager.Handle, ServerAppId);
+
+            var tcs = new TaskCompletionSource<MediaControllerError>();
+            string reqeustId = null;
+
+            EventHandler<CommandCompletedEventArgs> eventHandler = (s, e) =>
+            {
+                if (e.RequestId == reqeustId)
+                {
+                    tcs.TrySetResult(e.Result);
```

Comment:
You change return value from Task<int> to Task. 
and it means, it never fail to request. 
is it right?

Reply:
Exception will be thrown below if request is failed. Not return result code as you're commented.
Because it's not status value but error code.

(await tcs.Task).ThrowIfError("Failed to request command");

Reply:
Ok, i understand